plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetastoreAccessOperations.java

```diff
@@ -296,7 +297,7 @@ public void testSelectSystemTable()
 
         // This test should get updated if a new system table is added.
```

Comment:
> There's one commit that's different from CLA signer. Please update the correct author. If it was also worked on by someone else, please add them as "co-authors". [doc](https://docs.github.com/en/github-ae@latest/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-the-command-line)

Actually its an issue because I'm having two different GitHub account, will amend the author on that commit as well.
`git commit --amend --author= `

Reply:
@phd3 added the the `all_files` as well.